test/helper/integration.lua

```diff
@@ -5,4 +5,28 @@ local fio = require('fio')
 local shared = require('test.helper')
 local helper = {shared = shared }
 
+
+local root_url = '/'
+local function join_url(...)
+    local url = ''
+    for i, piece in ipairs({...}) do
+        if i == 1 then
+            url = piece
+        else
+            url = string.format('%s/%s', url, piece)
+        end
+    end
+    return url
+end
+
+function helper.path_to_storage(storage)
```

Comment:
What is `storage`? It looks like `storage_name`.

Moreover, I think that you can define `/kv` as a `base_endpoint` or smth like this in the test file directly (or `kv_base_endpoint` in helper)

Reply:
Yeah, really, there are no any storages are expected. So I can make common `base_enpoint` for all of tests